packages/backend/src/services/CsvService/CsvService.ts

```diff
@@ -443,6 +447,19 @@ export class CsvService extends BaseService {
             metricQueryWithDashboardFilters.additionalMetrics,
             metricQueryWithDashboardFilters.tableCalculations,
         );
+
+        // If a chart has been date zoomed, then its label has changed. We need to update the itemMap to reflect this in the CSV file
+        if (dateZoomGranularity) {
```

Comment:
_hack_ to replace the item with the new label 

For example

- user has chart in dashboard `created_month`
- applies date zoom day
- content of dimension remains the same except for: sql, label, and a few others
- we replace the `created_month` with the overriden date zoom dim in `itemMap` so that the CSV generated gets the overriden label (instead of Created Month, it's Created Day)

Reply:
We can remove `const itemsMap` and use `fields`.

Reply:
Ok great! I wondered if I would be missing something if `fields` were used. But glad you got that suggestion, nice